### PR TITLE
Adding Fastapi example

### PR DESCRIPTION
**Pull request by sagecodes**
## Description

Added a FastAPI example to integrations folder.

The commits in this pull request will 

## Changes

- Imperative commit or change title (commit ID) 
  
- two .py files for FastAPI example
- a Readme explaining example

## Related

Relates to organization/repo#number

Closes organization/repo#number

- [ ] I have reviewed the [Guidelines for Contributing](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md).

### CODE REVIEW

File: python/examples/integrations/fastapi/api.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# Set class names for prediction
+CLASS_NAMES = ["setosa", "versicolor", "virginica"]
+
+# Load model
+model = joblib.load('knn_model.pkl')
+
+# Set WhyLabs variables
+os.environ["WHYLABS_DEFAULT_ORG_ID"] = "org-qdcnk5"
+os.environ["WHYLABS_DEFAULT_DATASET_ID"] = "model-77"
+os.environ["WHYLABS_API_KEY"] = "NzhivaYyRM.j1di4X4maxSYkf85fwMOaxRusOh96fQSKp1ifRVCjTYsB7w4z4vpG"
```
Comment (murilommen) [inline python/examples/integrations/fastapi/api.py]: you should not commit api keys and secrets here

File: python/examples/integrations/fastapi/README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+### 5. View the logged data in WhyLabs
+Go to the [WhyLabs dashboard](https://app.whylabsapp.com/) and click on the project setup for logging. 
+
+Once profiles are written to WhyLabs they can be inspected, compared, and monitored for data quality and data drift.
+
+![detecting data drift with WhyLabs](https://whylabs.ai/_next/image?url=https%3A%2F%2Fcontent.whylabs.ai%2Fcontent%2Fimages%2F2022%2F11%2FFrame-6899--1-.png&w=3120&q=75)
+
+Learn more about how to use WhyLabs to monitor your ML models in production [here](https://docs.whylabsapp.com/).
+
+## A decoupled architecture approach to ML monitoring
```
Comment (murilommen) [inline python/examples/integrations/fastapi/README.md:136]: this was rephrased by Anthony with a more neutral language. Might copy from it https://docs.whylabs.ai/docs/fastapi-integration#decoupling-whylogs-from-fastapi

**Review by murilommen — CHANGES_REQUESTED** [review]
All in all looks good. Some changes are required. 
Do remember to update the [base README](https://github.com/whylabs/whylogs/blob/mainline/python/examples/README.md) under the model lifecycle section with an added line for FastAPI

File: python/examples/integrations/fastapi/api.py (hunk 1 above)
Comment (sagecodes) [reply python/examples/integrations/fastapi/api.py]: thanks for catching, I added back in for testing and forgot to remove! - removed

**Comment by murilommen** [comment]
another thing worth mentioning is that your code styling is not passing. you can make all checks by running: `make fix-format`, if you're on a ubuntu environment. to run that, you will need to have your environment properly setup. tomorrow I can help you further, in case you get stuck

**Comment by jamie256** [comment]
> another thing worth mentioning is that your code styling is not passing. you can make all checks by running: `make fix-format`, if you're on a ubuntu environment. to run that, you will need to have your environment properly setup. tomorrow I can help you further, in case you get stuck

whylogs CI uses pre-commit hooks to check a number of things in addition to the formatting, the following command will run the set of checks, and try to automatically fix things it can:

`make pre-commit`

**Review by murilommen — APPROVED** [review]

**Pull request merged**